frontend/src/views/omni/InstallationMedia/Steps/CloudProvider.vue

```diff
@@ -5,9 +5,138 @@ Use of this software is governed by the Business Source License
+    },
+    {
+      value: 'openstack',
+      label: 'OpenStack',
+      description: 'Runs on OpenStack virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/openstack',
+    },
+    {
+      value: 'vmware',
+      label: 'VMWare',
+      description: 'Runs on VMWare ESXi virtual machines',
+      documentation: '/platform-specific-installations/virtualized-platforms/vmware',
+    },
+    {
+      value: 'akamai',
+      label: 'Akamai',
+      description: 'Runs on Akamai Cloud (Linode) virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/akamai',
+    },
+    {
+      value: 'cloudstack',
+      label: 'Apache CloudStack',
+      description: 'Runs on Apache CloudStack virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/cloudstack',
+    },
+    {
+      value: 'hcloud',
+      label: 'Hetzner',
+      description: 'Runs on Hetzner virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/hetzner',
+    },
+    {
+      value: 'oracle',
+      label: 'Oracle Cloud',
+      description: 'Runs on Oracle Cloud virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/oracle',
+    },
+    {
+      value: 'upcloud',
+      label: 'UpCloud',
+      description: 'Runs on UpCloud virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/upcloud',
+    },
+    {
+      value: 'vultr',
+      label: 'Vultr',
+      description: 'Runs on Vultr Cloud Compute virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/vultr',
+    },
+    {
+      value: 'exoscale',
+      label: 'Exoscale',
+      description: 'Runs on Exoscale virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/exoscale',
+    },
+    {
+      value: 'opennebula',
+      label: 'OpenNebula',
+      description: 'Runs on OpenNebula virtual machines',
+      documentation: '/platform-specific-installations/virtualized-platforms/opennebula',
+    },
+    {
+      value: 'scaleway',
+      label: 'Scaleway',
+      description: 'Runs on Scaleway virtual machines',
+      documentation: '/platform-specific-installations/cloud-platforms/scaleway',
+    },
+  ].map((p) => ({
+    ...p,
+    documentation: getDocsLink('talos', p.documentation, {
+      talosVersion: formState.value.talosVersion,
+    }),
+  })),
+)
 </script>
 
-<template>Cloud Provider</template>
+<template>
+  <RadioGroup v-mode="formState.cloudPlatform" label="Cloud">
+    <RadioGroupOption v-for="platform in platforms" :key="platform.value" :value="platform.value">
+      {{ platform.label }}
+
+      <!-- prettier-ignore -->
+      <template #description>
+        {{ platform.description }} (<a class="text-primary-p3 underline" :href="platform.documentation">documentation</a>).
```

Comment:
I'd suggest to add `click.prevent` or something here to avoid selecting the radio button, when you just want to see the docs.